package.json

```diff
@@ -44,6 +44,7 @@
       "fixtures"
     ]
   },
+  "packageManager": "yarn@3.0.2",
```

Comment:
FYI: this is part of the new `corepack` feature coming to Node

https://nodejs.org/api/corepack.html

My understanding is that Node will ship "shims" for `yarn` and `pnpm` that delegate to a local install, so you don't need a global `yarn` executable to be installed manually that then defers to a local install; you'll have one already, much like how `npm` is _just there_ every time you install Node.